types/v56/types.go

```diff
@@ -1134,6 +1135,7 @@ type VMGeneralParams struct {
 	Name               string `xml:"Name,omitempty"`               // Name of VM
 	Description        string `xml:"Description,omitempty"`        // VM description
 	NeedsCustomization bool   `xml:"NeedsCustomization,omitempty"` // True if this VM needs guest customization
+	RegenerateBiosUuid bool   `xml:"RegenerateBiosUuid,omitempty"` // True if BIOS UUID of the virtual machine should be regenerated so that it is unique, and not the same as the source virtual machine's BIOS UUID.
```

Comment:
Is this being impacted by version bump or  is it for separate PR?

Reply:
Impact from version bump, nothing major - just introduced new field

Reply:
But is it being used anywhere yet (although I don't see in PR)? Just wondering if it is not changing the behavior as currently we are not regenerating it.

Reply:
We don't use it anywhere yet, just added to be up to date as we have such entity and new field arrived.